src/Bicep.LangServer/LangServerResources.resx

```diff
@@ -123,8 +123,11 @@
   <data name="DeploymentFailedWithExceptionMessage" xml:space="preserve">
     <value>Deployment failed for {0}. {1}</value>
   </data>
+  <data name="DeploymentFailedWithExceptionMessageAndLink" xml:space="preserve">
+    <value>Deployment failed for {0}. {1} Link to deployment available: {2}</value>
```

Comment:
```suggestion
    <value>Deployment failed for {0}.\n{1}View deployment in portal: {2}</value>
```

Reply:
Confer with Uche on exact wording, but I would at least get rid of "available".

Reply:
Another possibility:   
```
Details: <link>
```

Reply:
Actually, how about:
```md
See [details](link).
```
I tested, this works.  The raw link is rather ugly.

Reply:
@bhsubra I love Stephen's idea of See [details].

I would love for it to be: 

[Click to view deployment in portal](portal)

Thanks @StephenWeatherford!